### PR TITLE
MHP-2930-Create-Modal-For-Web-Dashboard-Update

### PR DESCRIPTION
**Pull request by TheNoodleMoose**
Made a new service specifically for this modal. I figured this would make removing it be an easier process.

### CODE REVIEW

**Comment by TheNoodleMoose** [comment]
@OzzieOrca I think you mentioned that the insights tab is going to become the new default tab, should I add that to this branch? Or should it be another ticket for a future date?

File: app/assets/javascripts/angular/services/insightsUpdateModalService.service.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+                    closeLabel: () => i18next.t('general.ok'),
+                    dismissLabel: () => null,
+                },
+                keyboard: false,
+                windowClass: 'pivot_theme',
+                backdrop: 'static',
+            });
+
+            try {
+                const modalResponse = await confirmModal.result;
```
Comment (OzzieOrca) [inline app/assets/javascripts/angular/services/insightsUpdateModalService.service.js]: You using this variable?

**Comment by OzzieOrca** [comment]
Ya it should become the default. Doesn't matter where you do it. Hopefully it's trivial.

File: app/assets/javascripts/angular/components/organizationOverview/organizationOverview.html [hunk 2]
```diff
@@ -7,11 +7,14 @@
...
             class="flex items-center justify-between br3 br--top bg-pivot-dark-blue"
         >
-            <div
-                class="pivot-white f3 ph3 pt3"
-                ui-sref="^.ministry.defaultTab({ orgId: $ctrl.org.id })"
-            >
-                {{ $ctrl.org.name }}
+            <div class="pivot-white f3 ph3 pt3">
+                <a
+                    class="pivot-white org-link"
```
Comment (TheNoodleMoose) [inline app/assets/javascripts/angular/components/organizationOverview/organizationOverview.html]: For some reason, I was trying to use the class no-underline(http://tachyons.io/components/links/dim-no-underline/index.html) but I think default scss was overriding it so I just made this class instead.

File: src/communityInsights/insights.tsx [hunk 3]
```diff
@@ -13,10 +13,21 @@ import Layout from './containers/Layout';
...
-const Insights = ({ orgId, authenticationService }: Props) => {
+const Insights = ({
+    orgId,
+    authenticationService,
+    insightsUpdateModalService,
+}: Props) => {
     const token = authenticationService.isTokenValid();
     const apolloClient = createApolloClient(token);
+    // TODO Remove once no longer needed
+    useEffect(() => {
```
Comment (TheNoodleMoose) [inline src/communityInsights/insights.tsx]: I didn't really like doing this, as it is running this check every time it renders but I couldn't use the service outside of the function.

Comment (OzzieOrca) [reply src/communityInsights/insights.tsx]: The `[]` dependencies array should only allow it to run once on mount. I think it's fine.

**Comment by TheNoodleMoose** [comment]
@OzzieOrca This one should wait to be merged to master until all the graphql stuff is finished right? Since we changed the default tab to the insights tab.

File: src/i18n/locales/en-US.ts [hunk 4]
```diff
@@ -1155,6 +1155,9 @@ export default {
         },
     },
     insights: {
+        insightsUpdateTitle: 'New Insights Dashboard',
+        insightsUpdateContent:
+            'Check out the new Insights dashboard to see your team’s steps of faith, interactions and challenges. Now you can see and celebrate what God is doing in your movement.',
```
Comment (OzzieOrca) [inline src/i18n/locales/en-US.ts]:
```suggestion
            'Check out the new Insights dashboard to see your team’s steps of faith, interactions and challenges. Now you can see and celebrate what God is doing in your community.',
```

Although I guess web still calls them `ministries`... Idk

**Comment by OzzieOrca** [comment]
I'm closing this... Sorry :( Someday users will get Insights.

**Pull request closed without merging**